t/data/04-complex_template.html

```diff
@@ -631,8 +631,8 @@
 <?lsmb END ?>
 <form action="<?lsmb script ?>" method="post">
 	<?lsmb PROCESS input element_data = {
-		type="hidden" 
-		name="entity_id" 
+		type="hidden"
+		name="entity_id"
```

Comment:
The space-cleanup can be part of a PR that preceeds this one, no?

Reply:
Fixed